src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -98,7 +98,8 @@
 
         return worldState.getBlock() instanceof IBuilderUndestroyable
                  || worldState.getBlock() == Blocks.BEDROCK
-                 || (info.getBlockInfo().getState().getBlock() instanceof AbstractBlockHut && handler.getWorldPos().equals(worldPos));
+                 || (info.getBlockInfo().getState().getBlock() instanceof AbstractBlockHut && handler.getWorldPos().equals(worldPos)
+                       && worldState.getBlock() instanceof AbstractBlockHut);
     };
```

Comment:
What exactly does this change?

Reply:
It makes sure that the anchor block with the AbstractBlockHut in the schematic is only ignored when there is an AbstractBuilding in the world. E.g. for a building request, it will ignore the position as before, but in a decoration request, if there needs to go a hut block there, the builder will place it there.